package.json

```diff
@@ -1,35 +1,66 @@
 {
```

Comment:
The Açaí standard is 4 space tab, it should be being enforced by the linter, we need to add the hook to lint fix it before commiting.